tests/pytests/unit/modules/virt/test_helpers.py

```diff
@@ -34,3 +34,11 @@ def assert_called(mock, condition):
     I know it's a simple XOR, but makes the tests easier to read
     """
     assert not condition and not mock.called or condition and mock.called
+
+
+def assertEqualUnit(actual, expected, unit="KiB"):
```

Comment:
Could we covert this name away from camel case? We want to dissociate these tests from unittest `assert<Something>`

Reply:
sure, I'll change those

Reply:
Done. I also added a commit to change the same thing for the remaining calls in the `tests/unit/modules/test_virt.py` file to ease future tests migration.